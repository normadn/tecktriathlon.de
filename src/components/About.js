import React from "react";

import BriefSummary from "./BriefSummary";
import SellingPoints from "./SellingPoints";

const About = () => (
  <section id="about">
    <BriefSummary />
    <SellingPoints />
  </section>
);

export default About;
