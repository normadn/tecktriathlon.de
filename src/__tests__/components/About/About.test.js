import React from "react";
import About from "../../../components/About";
import { shallow } from "enzyme";

describe("About", () => {
  it("renders correctly", () => {
    expect(shallow(<About />)).toMatchSnapshot();
  });
});
