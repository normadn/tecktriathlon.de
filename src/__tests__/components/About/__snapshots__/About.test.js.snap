// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About renders correctly 1`] = `
<section
  id="about"
>
  <BriefSummary />
  <SellingPoints />
</section>
`;
